Elements/src/Model.cs

```diff
@@ -192,7 +193,7 @@ public IEnumerable<T> AllElementsOfType<T>()
         /// <summary>
         /// Serialize the model to JSON.
         /// </summary>
-        public string ToJson(bool indent = false)
+        public string ToJson(bool indent = false, bool gatherSubElements = true)
```

Comment:
We did not previously pass through the `gatherSubElements` flag. It's a small optimization that a user can do to put things into the model themself, without recursive addition. We should support it at the level of the serialization API.

Reply:
👍